shadow-dom/reference-target/tentative/anchor.html

```diff
@@ -44,13 +44,40 @@
     <div id="popover-1" popover anchor="x-anchor-1">Popover content</div>
   </div>
 
+  <div>
+    <x-anchor-2 id="x-anchor-2"></x-anchor-2>
+    <div id="popover-2" popover anchor="x-anchor-2">Popover content</div>
+  </div>
+
+  <script>
+    const customAnchor = document.querySelector('x-anchor-2');
+    customAnchor.attachShadow({ mode: 'open', referencetarget: 'anchor' });
+    customAnchor.shadowRoot.innerHTML =`
+<style>
+div {
+width: 100px;
+height: 100px;
+}
+
+#anchor {
+background-color: yellow;
+}
+</style>
+
+<div></div>
+<div id="anchor"></div>`;
+  </script>
   <script>
-    test(function () {
-      const popover = document.getElementById("popover-1");
+    function testPopoverAnchor(id, name) {
+      test(function () {
+      const popover = document.getElementById(id);
       popover.showPopover();
       assert_equals(popover.offsetLeft, 100, "popover.offsetLeft");
       assert_equals(popover.offsetTop, 100, "popover.offsetTop");
-    }, "ShadowRoot ReferenceTarget works with anchor attribute.");
+      }, name);
+    }
+    testPopoverAnchor('popover-1', "ShadowRoot ReferenceTarget works with anchor attribute.");
+    testPopoverAnchor('popover-2', "ShadowRoot ReferenceTarget works with anchor attribute via options.");
```

Comment:
After fixing the above typo this test still fails with `assert_equals: popover.offsetTop expected 100 but got 300`.

I believe this is because the `<x-anchor>` from the first test is still in the DOM, pushing the contents for `<x-anchor-2>` down and resulting in a different offset.

One fix would be to remove the `<x-anchor>` as cleanup after the first test is done. Another fix would be to pass in the expected offsetTop as a parameter to `testPopoverAnchor`.